ci/jobs/scripts/workflow_hooks/team_notifications.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+from ci.praktika.info import Info
+from ci.praktika.gh import GH
+
+integrations_ecosystem_files = ["src/DataTypes"]
+
+
+def check():
+    info = Info()
+
+    changed_files = info.get_custom_data("changed_files")
```

Comment:
As far as I understand, the team will be notified of any change in any file in `src/DataTypes/*`. It might be too aggressive. We're interested in *new* data types added, which can be detected either by detecting *new* files in `src/DataTypes` directory (as per https://github.com/ClickHouse/ClickHouse/pull/81571#discussion_r2140017021) or on changes in `src/Core/TypeId.h` file